### PR TITLE
Feature/pmd depth cam

### PR DESCRIPTION
**Pull request by michaelpantic**
Simulates a PMD Monstar depthcam

### CODE REVIEW

File: rotors_description/urdf/pmd_monstar.xacro [hunk 1]
```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0"?>
+<robot name="$(arg mav_name)" xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find rotors_description)/urdf/component_snippets.xacro" />
+
+  <!-- Note that whenever possible, values from Realsense D435 were used.
+       See Intel Document 337029-004: Intel RealSense TM D400 Series Product Family Datasheet.
```
Comment (ffurrer) [inline rotors_description/urdf/pmd_monstar.xacro]: maybe update :)

File: rotors_description/urdf/pmd_monstar.xacro [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+  </xacro:macro>
+
+  <!-- Call depth sensor macro
+    Realsense values -
+      91.2 Deg Horizontal FOV
+      848x480 Optimal depth resolution
+        w 195 mm minimium-z depth
+      5 cm baseline.
+  -->
```
Comment (ffurrer) [inline rotors_description/urdf/pmd_monstar.xacro]: rm or change

File: rotors_description/urdf/pmd_monstar.xacro [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+      <origin xyz="0 0 0" rpy="${-pi/2} 0 ${-pi/2}" />
+      <parent link="${namespace}/camera_${camera_suffix}_link" />
+      <child link="${namespace}/camera_${camera_suffix}_optical_center_link" />
+      <limit upper="0" lower="0" effort="0" velocity="0" />
+    </joint>
+
+    <!-- Call rgb sensor macro-->
+
+    <gazebo reference="${namespace}/camera_${camera_suffix}_link">
+      <sensor type="depth" name="${namespace}_camera_{camera_suffix}">
```
Comment (kajabo) [inline rotors_description/urdf/pmd_monstar.xacro]: ${camera_suffix}

File: rotors_description/urdf/pmd_monstar.xacro [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+            <format>L8</format>
+            <width>${width}</width>
+            <height>${height}</height>
+          </image>
+          <clip>
+            <near>${min_range}</near>
+            <far>${max_range}</far>
+          </clip>
+        </camera>
+        <plugin name="${namespace}_camera_{camera_suffix}" filename="librotors_gazebo_noisydepth_plugin.so">
```
Comment (kajabo) [inline rotors_description/urdf/pmd_monstar.xacro]: ${camera_suffix}

**Comment by rikba** [comment]
Any update on this?

**Pull request merged**